client/src/main/scala/drt/client/services/handlers/ShiftsHandler.scala

```diff
@@ -23,8 +23,12 @@ class ShiftsHandler[M](viewMode: () => ViewMode, modelRW: ModelRW[M, Pot[ShiftAs
       val fixedPointsEffect = Effect(Future(GetShifts())).after(1 minute)
       log.info(s"Calling getShifts")
 
+      val maybeMillis = viewMode() match {
```

Comment:
this could probably benefit from a type annotation.

Reply:
yep